From 31e0cfc511eea1eac478223ad4c36e85b4172dbb Mon Sep 17 00:00:00 2001
Message-ID: <31e0cfc511eea1eac478223ad4c36e85b4172dbb.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 28 Oct 2023 04:28:44 +0100
Subject: [PATCH v3 26/66] gnu: go-github-com-google-safehtml: Move to (gnu
 packages golang-web).

* gnu/packages/golang.scm (go-github-com-google-safehtml): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: Ia1f14486f262d3695077c8e2f409f4928874c4f8
---
 gnu/packages/golang-web.scm | 26 ++++++++++++++++++++++++++
 gnu/packages/golang.scm     | 25 -------------------------
 2 files changed, 26 insertions(+), 25 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 117742dc07..ce95d1b467 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -8,6 +8,7 @@
 ;;; Copyright © 2022 Giacomo Leidi <goodoldpaul@autistici.org>
 ;;; Copyright © 2022 muradm <mail@muradm.net>
 ;;; Copyright © 2023 Hilton Chain <hako@ultrarare.space>
+;;; Copyright © 2023 Katherine Cox-Buday <cox.katherine.e@gmail.com>
 ;;; Copyright © 2023 Nicolas Graves <ngraves@ngraves.fr>
 ;;;
 ;;; This file is part of GNU Guix.
@@ -146,6 +147,31 @@ (define-public go-github-com-felixge-httpsnoop
 application's http.Handlers.")
     (license license:expat)))
 
+(define-public go-github-com-google-safehtml
+  (package
+    (name "go-github-com-google-safehtml")
+    (version "0.1.0")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/google/safehtml")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "0j2xjy8xrk9y9k6bqpvimj84i6hg1wwsyvwsb0axhmp49cmnrp86"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/google/safehtml"))
+    (propagated-inputs `(("go-golang-org-x-text" ,go-golang-org-x-text)))
+    (home-page "https://github.com/google/safehtml")
+    (synopsis "Safe HTML for Go")
+    (description
+     "Package safehtml provides immutable string-like types which represent
+values that are guaranteed to be safe, by construction or by escaping or
+sanitization, to use in various HTML contexts and with various DOM APIs.")
+    (license license:bsd-3)))
+
 (define-public go-github-com-gorilla-context
   (let ((commit "08b5f424b9271eedf6f9f0ce86cb9396ed337a42")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index f110da8d99..89aeead50a 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -12374,31 +12374,6 @@ (define-public go-github-com-google-go-jsonnet
 implementation.")
     (license license:asl2.0)))
 
-(define-public go-github-com-google-safehtml
-  (package
-    (name "go-github-com-google-safehtml")
-    (version "0.1.0")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/google/safehtml")
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "0j2xjy8xrk9y9k6bqpvimj84i6hg1wwsyvwsb0axhmp49cmnrp86"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/google/safehtml"))
-    (propagated-inputs `(("go-golang-org-x-text" ,go-golang-org-x-text)))
-    (home-page "https://github.com/google/safehtml")
-    (synopsis "Safe HTML for Go")
-    (description
-     "Package safehtml provides immutable string-like types which represent values
-that are guaranteed to be safe, by construction or by escaping or sanitization,
-to use in various HTML contexts and with various DOM APIs.")
-    (license license:bsd-3)))
-
 (define-public go-github-com-google-shlex
   (package
     (name "go-github-com-google-shlex")
-- 
2.41.0

